src/components/message/MessageText.jsx

```diff
@@ -1,26 +1,40 @@
 import React from "react";
+import connect from "react-redux/es/connect/connect";
 import parseEmoticons from "./parsers/parseEmoticons";
 import parseMedia from "./parsers/parseMedia";
 import parseFormatting from "./parsers/parseFormatting";
 import styled from "styled-components";
+import theme from "../../constants/theme";
 
 const MessageTextContainer = styled.div`
 	.emoticon {
 		max-height: 24px;
 	}
+	&.mention {
+		background-color: ${theme.mentionBackgroundColor ? theme.mentionBackgroundColor : '#faf2cc'};
```

Comment:
Move '#faf2cc' into the theme.js as the default (bottom block) vs having it here IMO. Then this can just say `background-color: ${theme.mentionBackgroundColor}`

Reply:
👍